schemas/Composer.json

```diff
@@ -25,6 +25,10 @@
         "description": "Whether this message it's a shared location.",
         "type": "boolean"
     },
+    "isPoll": {
```

Comment:
Does this property provide any value given there's a dedicated event for starting a poll?

Reply:
I asked the same but @VolkerJunginger said they want the composer event as well, I'll post his answer here:

> No, it is not redundant from an analytics point of view.
> We need the composer Event to count "no of messages send" overall.
> We need the poll event to get data for the poll

Reply:
Ok, that makes sense, thinking out loud here: it feels like we're doing it wrong in that case. having `isPoll`, `locationType` as separate properties isn't very scalable. Because it sounds like we also want an `isImage`, `isVideo` etc if we want all messages sent.

Perhaps we should have a `messageType` enum here with initially, `Text`, `Poll`, `LocationPin` and `LocationUser` and then in the future this could be expanded to include other types of message content?

@VolkerJunginger does that sound reasonable to you (and would you have an issue if this caused any old `locationType` properties to be misaligned)?

Reply:
Given what's been discussed in the analytics room I'll change this as suggested. Though it'll have to wait for me to be back from vacation (~1week).